packages/uiweb/src/lib/components/notification/index.tsx

```diff
@@ -34,10 +36,21 @@ export type NotificationItemProps = {
 
 
 type ContainerDataType = {
-  cta?: boolean;
   timestamp?: string;
 };
 
+type ContentSectionDataType = {
+  cta?: boolean;
+};
+
+type ChannelTitleDataType = {
+  cta?: boolean;
```

Comment:
define only one type

Reply:
I'll change this.

Reply:
@madhurg-epns did you fix this, i cannot see it